ambry-commons/src/main/java/com.github.ambry.commons/ResponseHandler.java

```diff
@@ -109,6 +110,8 @@ private void onException(ReplicaId replicaId, Exception e) {
    * {@link ServerErrorCode}.
    */
   public void onEvent(ReplicaId replicaId, Object event) {
+    if(replicaId instanceof CloudReplica)
+      return;
```

Comment:
There is no backoff when replication with cloudReplica has an exception? (I guess it is based on Helix will notify which is next available cloud replica)

Reply:
Yes.